app/code/Magento/MediaGalleryCatalogUi/Test/Mftf/Test/AdminMediaGalleryCatalogUiVerifyCategoryGridPageTest.xml

```diff
@@ -26,6 +26,13 @@
             <deleteData createDataKey="category" stepKey="deleteCategory"/>
         </after>
         <actionGroup ref="AdminOpenCategoryGridPageActionGroup" stepKey="openCategoryPage"/>
-        <actionGroup ref="AdminAssertCategoryGridPageDetailsActionGroup" stepKey="assertCategoryGridPageRendered"/>
+
+        <actionGroup ref="AdminSearchCategoryGridPageByCategoryNameActionGroup" stepKey="searchByCategoryName">
+            <argument name="categoryName" value="$$category.name$$"/>
+        </actionGroup>
+
+        <actionGroup ref="AssertAdminCategoryGridPageDetailsActionGroup" stepKey="assertCategoryGridPageRendered">
+            <argument name="category" value="$$category$$"/>
+        </actionGroup>
```

Comment:
It would be good to revert the grid state with `ResetAdminDataGridToDefaultViewActionGroup`